packages/multi-select-combo-box/test/selecting-items.test.js

```diff
@@ -110,6 +118,14 @@ describe('selecting items', () => {
       comboBox.clear();
       expect(comboBox.selectedItems).to.deep.equal([]);
     });
+
+    it('should keep the filter and input value when committing an invalid option', async () => {
```

Comment:
Added the test from #7072 and updated the test in the `keep filter` suite to also verify that the filter is not reset.